l10n_nl_rgs/data/account_fiscal_position_tax_template.xml

```diff
@@ -55,22 +55,22 @@
         <field name="tax_dest_id" ref="btw_X3"/>
     </record>
     <!-- VAT on buying from outside the EU -->
-    <record id="position_tax_extracom_9" model="account.fiscal.position.tax.template">
+    <record id="position_tax_extracom_9_buy" model="account.fiscal.position.tax.template">
```

Comment:
These were overwritten by Odoo. Perhaps this should be checked. I have given them a unique identifier for now, and I don't see it being referenced in code anywhere.

Reply:
@MrGigSolutions , please keep these IDs as it is. It wouldn't get overwriiten as the ids are a combination of the module name and the ID itself